webanno-api-annotation/src/main/java/de/tudarmstadt/ukp/clarin/webanno/api/annotation/rendering/model/VSpan.java

```diff
@@ -40,20 +41,20 @@
     public VSpan(AnnotationLayer aLayer, AnnotationFS aFS, String aType, VRange aOffsets,
             Map<String, String> aFeatures)
     {
-        this(aLayer, new VID(getAddr(aFS)), aType, asList(aOffsets), aFeatures);
+        this(aLayer, new VID(getAddr(aFS)), aType, Collections.singletonList(aOffsets), aFeatures);
```

Comment:
Why is `Collections.singletonList` better than `(Collections.)asList`? asList is at least shorter ;)